interpreter/interpreter_test.go

```diff
@@ -847,7 +847,7 @@ func TestInterpreter_CustomTypeProvider(t *testing.T) {
 	i, _ := interp.NewInterpretable(checked)
 	vars := EmptyActivation()
 	result := i.Eval(vars)
-	if !types.IsUnknownOrError(result) {
+	if !types.IsUnknown(result) {
```

Comment:
Since we want the result to be an error in the future, I was suggesting leaving the test as IsUnknownOrError but changing the message to "wanted error" or "wanted unknown or error".

Reply:
I've updated the test condition with a TODO since it reflects the current state of the code and will have to be updated in order to address Issue #190.

Reply:
I would have been happy with the unknown-or-error check, with a note to clean up after the fix, but this is okay for now.